textworld/gym/core.py

```diff
@@ -0,0 +1,54 @@
+from typing import List, Mapping, Any, Optional
+
+from textworld import EnvInfos
+
+
+class Agent:
+    """ Interface for any agent playing TextWorld games. """
+
+    @property
+    def infos_to_request(self) -> EnvInfos:
```

Comment:
This doesn't match the current contest API (`select_additional_infos()`), I guess that's okay?

Reply:
Yeah, I didn't like `select_additional_infos` so I'm changing it now :P. That said, this abstract agent class is not used yet.

Reply:
Yeah sounds good.  Like you say, there's no framework users yet.  We can always make the contest compatible with both agent APIs.